src/components/ChangeProfileDataModal/ChangeProfileDataModal.tsx

```diff
@@ -29,21 +21,12 @@ const ChangeProfileDataModal: FC<ChangeProfileDataModalProps> = ({ user }) => {
   });
 
   const changeData = () => {
-    fetch(`${UrlSite.URL}/user/profile`, {
-      credentials: 'include',
-      method: 'PUT',
-      headers: {
-        Accept: 'application/json',
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify(data),
-    }).then(() => {
-      location.reload();
-    });
+    updateProfileData(data)(dispatch);
```

Comment:
Thunks используются не совсем так
```suggestion
    dispatch(updateProfileData(data));
```